mc/gui/breathing_phrase_list_dock.py

```diff
@@ -210,7 +210,8 @@ class EditDialog(QtWidgets.QDialog):
     def __init__(self, i_parent=None):
         super(EditDialog, self).__init__(i_parent)
 
-        assert mc.mc_global.active_phrase_id_it != mc.mc_global.NO_PHRASE_SELECTED_INT
+        if mc.mc_global.active_phrase_id_it == mc.mc_global.NO_PHRASE_SELECTED_INT:
+            mc.mc_global.active_phrase_id_it = 1   
```

Comment:
Thank you for this idea, i've also been thinking of something like this. Not sure about the approach, what happens if there are no entries in the list?